docs/ref/settings.txt

```diff
@@ -146,7 +146,7 @@ The :setting:`APPEND_SLASH` setting is only used if
 CACHES
 ------
 
-Default::
```

Comment:
Your suggestion is fine. I don't see a need for "doc-wide consistency" because single line and multi-line examples have different syntaxes. A single colon appears in the rendered version of the docs in both cases, so that's what is most important.